packages/marshal/src/marshal.js

```diff
@@ -349,8 +356,8 @@ function assertCanBeRemotable(val) {
   const keys = ownKeys(descs); // enumerable-and-not, string-or-Symbol
   keys.forEach(key => {
     assert(
-      // @ts-ignore
-      !('get' in descs[key]),
+      // Typecast needed due to https://github.com/microsoft/TypeScript/issues/1863
+      !('get' in descs[/** @type {string} */ (key)]),
```

Comment:
@warner I thought I saw something from you saying that you tried this cast and it didn't work for you. Would be good to understand why vs why it works for @michaelfig . Possibly I screwed something up in relaying the cast syntax as I remembered it? Attn @Chris-Hibbert also on that thread.

Reply:
> Possibly I screwed something up in relaying the cast syntax as I remembered it?

Yeah, you had said `/** type{string} */` which didn't parse.

Don't worry about it, though.